src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/NewAzureVirtualNetworkGatewayCommand.cs

```diff
@@ -216,9 +219,25 @@ private PSVirtualNetworkGateway CreateVirtualNetworkGateway()
                 }
             }
 
-            if (this.EnableActiveActiveFeature.IsPresent && !vnetGateway.Sku.Tier.Equals(MNM.VirtualNetworkGatewaySkuTier.HighPerformance))
+            if (this.EnableActiveActiveFeature.IsPresent 
+                && !vnetGateway.Sku.Tier.Equals(MNM.VirtualNetworkGatewaySkuTier.HighPerformance)
+                && !vnetGateway.Sku.Tier.Equals(MNM.VirtualNetworkGatewaySkuTier.VpnGw1)
+                && !vnetGateway.Sku.Tier.Equals(MNM.VirtualNetworkGatewaySkuTier.VpnGw2)
+                && !vnetGateway.Sku.Tier.Equals(MNM.VirtualNetworkGatewaySkuTier.VpnGw3))
+            {
+                string errorMessage = string.Format(
```

Comment:
dont have any validations here. make sure the user can see the server error message